src/styles/index.scss

```diff
@@ -1,3 +1,69 @@
+@mixin center {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), mixins should be used for groups of CSS properties that are used together in various places to avoid code duplication. The `center` mixin is a good example of this, but consider if absolute positioning is the best strategy for centering elements as it removes them from the normal document flow. Flexbox or Grid could be alternatives that provide more flexibility and maintain document flow.